.github/workflows/go.yml

```
@@ -24,6 +24,11 @@ jobs:
     - name: Get dependencies
       run: go mod download
 
+    - name: Lint code
+      uses: golangci/golangci-lint-action@v2
+      with:
+        version: v1.29
```

Any reason not to use 1.32.0?
```suggestion
        version: v1.32
```